test/mainController.test.ts

```diff
@@ -205,18 +205,18 @@ suite('MainController Tests', () => {
     test('onNewQuery should call the new query and new connection' , () => {
 
```

Comment:
No actual test changes? They all pass? That signifies a pretty large gap in the test matrix if so. Do you have any plans on what you'd like to do around that given the renewed focus on fundamentals and quality?